ex2.sh

```diff
@@ -0,0 +1,14 @@
+ls -S
+a to present the context of the largest file#
+cat /etc/services
+b to find the largest file#
+ls -S /etc/ | head -1
+ls -S -r
+a to find the smallest file#
+ls -S /etc/ | tail -1
+now present the context of file#
+cat /etc/motd
+mkdir -p grandpa/father/son
+bc
+bc -q
+cal 2023

```

Comment:
They task is to write the year, this should be done dynamically, next year I want to run the project and see the year 2024, try to think how it can be done. try read about it here: https://www.tutorialkart.com/bash-shell-scripting/bash-current-year/